language-sql.md

```diff
@@ -5,3 +5,10 @@
 ### Minimum Requirements
 
 MySQL keywords in uppercase.
+
+### Database table naming convention
+
+Prefix application name to table name. E.g.: `app1_users`, `app2_log_meta`.
+
+If the table is for multiple applications (normally this should be avoided), it's ok to leave it without the application name prefix.
```

Comment:
I see, thanks. :) Would it be worth it to link to that URL in the README.md itself? (This is not a real question, mostly a suggestion)